YololCompetition/Services/Challenge/DbChallenges.cs

```diff
@@ -90,7 +90,8 @@ DbCommand PrepareQuery(IDatabase db)
             if (pending == null)
                 return null;
 
-            var endTime = (DateTime.UtcNow + TimeSpan.FromHours(_config.ChallengeDurationHours)).UnixTimestamp();
+            // End time of the challenge should be 3 days after the start time of today to account for if Referee was down on the day it was supposed to start the challenge.
```

Comment:
> should be 3 days

This comment doesn't seem right?

Reply:
This is intentional. Same reason why if it has been over 24 hours when referee is restarted it will start a challenge regardless of what time it is.